server/app.js

```diff
@@ -6,8 +6,9 @@ var logger = require('morgan');
 var cors = require('cors');
 
 var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
-var loginRouter = require('./routes/login');
+var userRouter = require('./routes/user');
+//var loginRouter = require('./routes/login');
```

Comment:
login.js doesn't seem to be used anywhere so it should be deleted. Along with all lines commented out